studyzen/appsettings.Development.json

```
@@ -1,8 +1,20 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
-    }
+  "Serilog": {
+    "Using": [
+      "Serilog.Sinks.Console",
+      "Serilog.Sinks.File"
+    ],
+    "WriteTo": [
+      {
+        "Name": "Console"
+      },
+      {
+        "Name": "File",
+        "Args": {
+          "path": "logs/studyzen-.log",
```

```suggestion
          "path": "%PROGRAMDATA%/StudyZen/Logs/studyzen-.log"
```

I think it would be nice to store logs in `ProgramData` directory, so we have everything in one place and have easy access: https://stackoverflow.com/questions/67983560/where-to-put-logs. Currently `GenericRepository` uses `Environment.SpecialFolder.ApplicationData`, which is AppData/Roaming, but I think it would be more appropriate to use `ProgramData`, since technically we'll have one server running that every client communicates with. If you decide to go this route, I think we should change to `Environment.SpecialFolder.ApplicationData` in `GenericRepository` to `Environment.SpecialFolder.CommonApplicationData`